backend/python/app/tests/test_dummy.py

```
@@ -0,0 +1,6 @@
+# This is a dummy test to ensure the CI pipeline is functional.
+# Remove this file when you implement real backend functionality.
+
+
+def test_dummy_ci_check():
+    assert 1 + 1 == 2
```

Ruff (0.12.2)

Use of `assert` detected

(S101)